### PR TITLE
test: adjust mock login

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] fix mock login in e2e

### <samp>🤖 Generated by Copilot at c6a2e1a</samp>

The pull request updates the e2e-login page and the Cypress tests to use the new Vue 3 composition API and a mock account for testing. It adds assertions to check that the mock account is created and stored in localStorage before running the tests.

### <samp>🤖 Generated by Copilot at c6a2e1a</samp>

> _`<script setup>` now_
> _mocks account for testing - but_
> _waits a bit, autumn_

### CODE REVIEW

File: pages/e2e-login.vue [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
+    localStorage.setItem(
+      'identity',
+      JSON.stringify({
+        auth: { address: account, balance: {}, tokens: {} },
+        identities: {},
+      })
+    )
+
+    mockAddress.value = true
+  }, 1000)
```
Comment (roiLeo) [inline pages/e2e-login.vue:38]: weird timeout but ok

**Review by roiLeo — APPROVED** [review]
✅ code lgtm 🧪

File: pages/e2e-login.vue (hunk 1 above)
Comment (preschian) [reply pages/e2e-login.vue]: removed

**Comment by preschian** [comment]
will check reviewdog in seperate PR

**Review by roiLeo — APPROVED** [review]

**Pull request merged**